src/ExtMongoDbResourceManager.php

```diff
@@ -51,8 +51,7 @@ public function setResource($id, $resource)
     {
         if ($resource instanceof Collection) {
             $this->resources[$id] = [
-                'db'                  => (string) $resource->db,
-                'db_instance'         => $resource->db,
```

Comment:
I have no idea what `db` was in the past, but now it's not alive anymore :shrug: